api/app.py

```diff
@@ -5,14 +5,14 @@
 from api.routes.onboarding import onboarding_bp
 from api.routes.stokvel_info import stokvel_info_bp
 from api.routes.user_info import user_info_bp
-from api.routes.whatsapp_controller import whatsapp_bp
+# from api.routes.whatsapp_controller import whatsapp_bp
 
 app = Flask(__name__)
 
 app.register_blueprint(join_stokvel_bp)
 app.register_blueprint(onboarding_bp)
 app.register_blueprint(user_info_bp)
-app.register_blueprint(whatsapp_bp)
+# app.register_blueprint(whatsapp_bp)
```

Comment:
Will have to remove the comments when pushing into main. I will send you guys the credentials for the Twilio client so that you dont get the twilio error